src/manpage/generator.ml

```diff
@@ -304,6 +313,55 @@ and inline (l : Inline.t) =
       | Math s -> math s ++ inline rest
       | Raw_markup t -> raw_markup t ++ inline rest)
 
+let table pp { Table.data; align } =
+  let sep = '\t' in
+  let alignment =
+    let alignment =
+      match align with
+      | Some align ->
+          List.map
+            (function
+              (* Since we are enclosing cells in text blocks, the alignment has
+                 no effect on the content of a sufficiently big cell, for some
+                 reason... (see the markup test in generators)
+
+                 One solution would be to use the [m] column specifier to apply
+                 a macro to the text blocks of the columns. Those macros would
+                 be [lj], [ce] or [rj], which define alignment. However, this
+                 breaks both the alignment for small table cells, and the
+                 largeness of columns. For the records, it woulb be:
+
+                 {[
+                   | Some `Left -> "lmlj"
+                   | Some `Center -> "cmce"
+                   | Some `Right -> "rmrj"
+                   | None -> "l"
+                  ]} *)
+              | Some `Left -> "l"
+              | Some `Center -> "c"
+              | Some `Right -> "r"
+              | None -> "l")
```

Comment:
I couldn't find a ref for the manpage syntax, is there no other choice for the default case (like keeping it empty) ?

Reply:
The ref for tables in the gtroff format is https://manpages.debian.org/testing/groff-base/tbl.1.en.html and https://doc.cat-v.org/unix/v10/10thEdMan/tbl.pdf

I could not find a "not specified" alignment. Moreover, the number of alignment _has to match_ the number of columns, otherwise the extra columns do not get rendered...

Reply:
Thanks! The default alignment seems to be left in the doc indeed.